apps/web/screens/IcelandicGovernmentInstitutionVacancies/IcelandicGovernmentInstitutionVacanciesList.tsx

```diff
@@ -204,8 +227,109 @@ const IcelandicGovernmentInstitutionVacanciesList: Screen<
     setSearchTerm('')
   }
 
+  const renderFilterTags = (
+    filterType: 'fieldOfWork' | 'location' | 'institution',
+    filters: string[],
+  ) => {
+    return filters.map((filter) => (
+      <Tag
```

Comment:
I wonder for consistency if using the `<FilterTag />` component instead might make sense (might even make sense to move it to the island-ui library at some point.